drivers/storage/ide/pciidex/pciidex.spec

```
@@ -1,3 +1,4 @@
+@ varargs PciIdeXDebugPrint(long ptr)
```

```suggestion
@ varargs PciIdeXDebugPrint(long str)
```
Sorry, missed it the first time.